src/Integration/Telemetry/Payload/TelemetryPayload.cs

```diff
@@ -72,57 +73,8 @@ public sealed class TelemetryPayload
 
         [JsonProperty("taint_vulnerabilities")]
         public TaintVulnerabilities TaintVulnerabilities { get; set; }
-    }
-
-    public sealed class IdeVersionInformation
```

Comment:
Cosmetics: I placed these classes in other files, all under "Payload" folder. It started to get a bit much.

Reply:
Please make this kind of change separately going forwards. It just adds noise and makes the "real" changes harder to review.